_assets/js/dist/accordion-compiled.js

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
You might consider minifying for production at some point.

Reply:
Oh dang, thank you for catching this! It should have been minifying already. This shouldn't even have been tracked...

Reply:
@billyhunt The JS compiled assets should be removed now.